assets/src/edit-story/components/helpCenter/toggle/index.js

```diff
@@ -17,62 +17,19 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 import { __, _n, sprintf } from '@web-stories-wp/i18n';
 
 /**
  * Internal dependencies
  */
-import {
-  BUTTON_TYPES,
-  BUTTON_VARIANTS,
-  BUTTON_SIZES,
-  Icons,
-  NotificationBubble,
-  Button as dsButton,
-  BEZIER,
-} from '../../../../design-system';
+import { Icons } from '../../../../design-system';
+import { ToggleButton } from '../../toggleButton';
 
-const Button = styled(dsButton)`
-  height: 36px;
-  border: 1px solid ${({ theme }) => theme.colors.border.defaultNormal};
-  padding: 8px;
-  color: ${({ theme }) => theme.colors.fg.primary};
-
-  ${({ isOpen, theme }) =>
-    isOpen &&
-    css`
-      border-color: ${theme.colors.bg.secondary};
-      background-color: ${theme.colors.bg.secondary};
-    `}
-`;
-
-const Label = styled.span`
-  display: block;
-  line-height: 20px;
-  text-align: left;
-`;
-
-const NotificationWrapper = styled.div`
-  margin: -2px 0 -2px 20px;
-`;
-
-const Chevron = styled(Icons.ChevronUpSmall)`
-  display: block;
-  margin: -6px 0 -6px 16px;
-  width: 32px;
+const MainIcon = styled(Icons.QuestionMarkOutline)`
   height: 32px;
-  transform-origin: 50% 50%;
-  transform: rotate(${({ $isOpen }) => ($isOpen ? 0 : 180)}deg);
-  transition: transform 300ms ${BEZIER.default};
-
-  @media ${({ theme }) => theme.breakpoint.mobile} {
-    ${({ hasNotifications }) =>
-      hasNotifications &&
-      css`
-        display: none;
-      `}
-  }
+  width: auto;
+  display: block;
```

Comment:
💭 I don't think we need to change this, but another option could be to add a container in the shared toggle that has a set height of 32 px. Passing any icon into it should fill that space exactly too

Reply:
I'm refactoring the popups to share code better, I'll do that there - this one I want to get through QA to prevent it from blocking us. That's a good idea, i like it!